mica-webapp/src/main/resources/_templates/network.ftl

```diff
@@ -182,7 +182,22 @@
                     <ul class="nav nav-tabs">
                       <li class="nav-item" v-for="(option, index) in chartOptions" v-bind:key="option.id"><a href="" class="nav-link" @click.prevent="onTabClick(index)">{{option.title}}</a></li>
                     </ul>
-                    <graphics-result v-bind:chart-options="chartOptions" v-bind:hide-header="true"></graphics-result>
```

Comment:
Could the `graphics-result` component be modified instead? (so the template does not change)

Reply:
ok